Integrations/SplunkPy/SplunkPy.yml

```diff
@@ -259,8 +281,61 @@ script:
     - contextPath: Splunk.Raw.Parsed
       description: The raw event data (parsed).
       type: unknown
+  - arguments:
+    - default: false
+      description: |-
+        Event payload key-value. Value can be a string or a JSON object.
+        JSON example: {"event": {"message":"Access log test message"}}
+        String example: "event": "Access log test message."
+      isArray: false
+      name: event
+      required: true
+      secret: false
```

Comment:
By this description, it seems like a valid run of the command could be: `!splunk-submit-event-hec {"event": {"message":"Access log test message"}}`. It is not correct right?

Reply:
According to the docs, you can send it both ways